content/test-two.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: some random markdown post
-date: 2021-08-18
-author: manusia_bernapas
-desc: some random post for testing purpose only, 何もないだよ
+title: Your Title Goes Here
+desc: The post description. Might be somethinig that would interest the reader in less than 100 words.
+author: Reinaldy Rafli
+github: aldy505
+twitter: aldy505
```

Comment:
did you fkn stalked me? lmao

Reply:
ngga om, awalnya cuma asal nebak, eh ternyata bener xixixi